spec/lib/grape/middleware/logger_spec.rb

```diff
@@ -100,6 +106,13 @@
       it 'defaults to the the standard Logger' do
         expect(subject.logger).to be_a(Logger)
       end
+
+      it 'defaults to Rails.logger if is set' do
+        rails_logger = double("rails_logger")
+        allow(Rails).to receive(:logger).and_return(rails_logger)
+
+        expect(subject.logger).to eq(rails_logger)
+      end
```

Comment:
@ridiculous of course, I was trying to do something like this, yet, outside the `rspec` my tests works fine. Opening `$ irb` and hand-testing, but for some reason when testing with `rspec` I was just getting some pretty bizarre errors of uninitialised constant `Rails` in the logger.   

Anyway I'm going to improve those tests.